source/indexes.txt

```diff
@@ -12,6 +12,26 @@ Manage Indexes
    :depth: 1
    :class: singlecol
 
+Indexes are special data structures that store a small portion of a 
+collection’s data set in an easy-to-traverse form. Indexes support the 
+efficient execution of queries in MongoDB.
+
+To improve query performance, build indexes on fields that appear often 
+in queries and for all operations that sort by a field. Queries on the 
+indexed field use the index to limit the number of documents that must 
+be scanned.
```

Comment:
So in the first sentence we talk about queries and sort operations. The next sentence explains why you should build an index for queries. Maybe should add a sentence or tweak the first to explain why you want one for sort operations?